Build/start.sh

```diff
@@ -54,13 +55,18 @@ fi
 sed -i 's/renesas \\/&\n  ${TOPDIR}\/..\/meta-mistysom \\/' ${WORK}/build/conf/bblayers.conf
 
 # add dunfell compatibility to layers wehre they're missing to avoid WARNING
-echo "LAYERSERIES_COMPAT_qt5-layer = \"dunfell\"" >> ${YOCTODIR}/meta-qt5/conf/layer.conf
-echo "LAYERSERIES_COMPAT_rz-features = \"dunfell\"" >> ${YOCTODIR}/meta-rz-features/conf/layer.conf 
+echo "LAYERSERIES_COMPAT_qt5-layer = \"dunfell\"" >> ${WORK}/meta-qt5/conf/layer.conf
+echo "LAYERSERIES_COMPAT_rz-features = \"dunfell\"" >> ${WORK}/meta-rz-features/conf/layer.conf 
+
+#set permissions for downloads & sstate-cache directories
+echo ${PASSWD} | sudo -S chmod -R 777 ${WORK}/build/downloads
+echo ${PASSWD} | sudo -S chmod -R 777 ${WORK}/build/sstate-cache
```

Comment:
I feel very uncomfortable about this.
Shouldn't the permissions be set outside of the container?

Reply:
I see what you're saying, this is just further automating/simplifying things. I guess `777` could/should be replaced by `+w` instead, would be okay then? Or are you stricly refusing to set permissions from the container? This will simplify  dealing with permissions but again, I see your point! By passing the known sudo password from within the container, we essentially modify the host directory permissions....

Reply:
optimized permission setting by host user and added check and print descriptive message on failure (should avoid being able to start the container without having access to the required directories". Also reduced the permissions of `OUTDIR` from `777` to `+w`